web_src/js/features/citation.js

```diff
@@ -0,0 +1,69 @@
+import $ from 'jquery';
+
+const {pageData} = window.config;
+
+const initInputCitationValue = async () => {
+  await import(/* webpackChunkName: "plugin-software-formats" */'@citation-js/plugin-software-formats');
+  await import(/* webpackChunkName: "plugin-bibtex" */'@citation-js/plugin-bibtex');
+  await import(/* webpackChunkName: "citation-js" */'citation-js').then((module) => {
```

Comment:
1. Would still suggest `Promise.all` format to import all modules concurrently, as the current serialized import will be slower.
2. Can we import `Cite` from `@citation-js/core`, replacing legacy `citation-js`?